packages/harmony/src/components/input/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,206 @@
+import { forwardRef, type ComponentPropsWithoutRef } from 'react'
+
+import cn from 'classnames'
+
+import layoutStyles from 'components/layout/layout.module.css'
+import { IconComponent, Text } from 'components/typography'
+
+import { HelperText } from './HelperText'
+import styles from './TextInput.module.css'
+import { useFocusState } from './useFocusState'
+
+export enum TextInputSize {
+  SMALL = 'small',
+  DEFAULT = 'default'
+}
```

Comment:
Previously there was a "large" size but design wants to stick to just 2 moving forward (the large was only like a few pixels bigger anyways)